docs/source/actions/custom-action.rst

```diff
@@ -13,3 +13,45 @@ the `run_action` method.
         def run_action(self):
             # Include here the logic of your action
             # (...)
+
+
+Fallback Actions
+================
+
+When creating your own custom actions, you can also add a fallback action to run if
+an action throws an unhandled exception. To do this, add a fallback attribute to
+your custom action.
+
+.. code-block:: python
+
+    from spidermon.core.actions import Action
+
+    class MyFallbackAction(Action):
+        def run_action(self):
+            # Include here the logic of your action
+            # Runs if MyCustomAction().run_action() throws an unhandled exception
+            # (...)
+
+    class MyCustomAction(Action):
+        fallback = MyFallbackAction()
```

Comment:
**question** What happens if I have a fallback action that has a fallback action that is the initial action?

```
    class MyFallbackAction(Action):
        fallback = MyCustomAction()

        def run_action(self):
            ...

    class MyCustomAction(Action):
        fallback = MyFallbackAction()

        def run_action(self):
            ...
```

Reply:
You can't! `fallback = MyCustomAction()` in `MyFallbackAction` will give a name error because `MyCustomAction` isn't defined yet.